curriculum/challenges/english/10-coding-interview-prep/rosetta-code/straddling-checkerboard.md

```diff
@@ -8,7 +8,11 @@ dashedName: straddling-checkerboard
 
 # --description--
 
-Implement functions to encrypt and decrypt a message using the [straddling checkerboard](https://en.wikipedia.org/wiki/Straddling_checkerboard) method. The functions will take a string and an array as parameters. The array has 3 strings representing the 3 rows of the checkerboard. The output will be a series of decimal digits. Numbers should be encrypted by inserting the escape character before each digit, then including the digit unencrypted. This should be reversed for decryption.
+Implement functions to encrypt and decrypt a message using the **straddling checkerboard** method.
+
+- The **functions** will take a <em>string</em> and an <em>array</em> as parameters.
```

Comment:
should the word functions be plural in this case?

Reply:
I thought functions refer to the seed content

```js
function straddle(message, alphabet) {

}

function unstraddle(message, alphabet) {

}
```

If I am misunderstanding, I am happy to fix it.